src/generators/css-filter.ts

```diff
@@ -63,7 +76,7 @@ export function cssFilterStyleheetTemplate(filterValue: string, reverseFilterVal
     });
 
     if (!frameURL) {
-        const [r, g, b] = applyColorMatrix([255, 255, 255], createFilterMatrix(config));
+        const [r, g, b] = hasChromiumIssue501582() ? applyColorMatrix([0, 0, 0], createFilterMatrix(config)) :applyColorMatrix([255, 255, 255], createFilterMatrix(config));
```

Comment:
Let's make code more clear. Make `applyColorMatrix` be called only once. Move array and filter matrix into separate variables.